### PR TITLE
Add Coalesce

### PR DESCRIPTION
**Pull request by eliascarv**
This PR adds the Coalesce transformation. The `Coalesce(default)` transformation replaces all missing values ​​with a default value.

### CODE REVIEW

File: src/transforms.jl [hunk 1]
```diff
@@ -216,6 +216,7 @@ include("transforms/filter.jl")
 include("transforms/rename.jl")
 include("transforms/identity.jl")
 include("transforms/center.jl")
+include("transforms/coalesce.jl")
```
Comment (juliohm) [inline src/transforms.jl:219]: Same here, before identity.jl

File: src/transforms/coalesce.jl [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+struct Coalesce{T} <: Colwise
```
Comment (juliohm) [inline src/transforms/coalesce.jl:5]: Missing docstring

File: src/transforms/coalesce.jl [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  default::T
```
Comment (juliohm) [inline src/transforms/coalesce.jl:6]: Rename the field to `value`.

**Review by juliohm — CHANGES_REQUESTED** [review]

File: src/transforms/coalesce.jl (hunk 3 above)
Comment (eliascarv) [reply src/transforms/coalesce.jl]: Made in [76d50d3](https://github.com/JuliaML/TableTransforms.jl/pull/40/commits/76d50d3a5bf0473adec685a23b83c0424881908c)

File: src/transforms/coalesce.jl (hunk 2 above)
Comment (eliascarv) [reply src/transforms/coalesce.jl:10]: Made in [76d50d3](https://github.com/JuliaML/TableTransforms.jl/pull/40/commits/76d50d3a5bf0473adec685a23b83c0424881908c)

File: src/transforms.jl (hunk 1 above)
Comment (eliascarv) [reply src/transforms.jl]: Made in [76d50d3](https://github.com/JuliaML/TableTransforms.jl/pull/40/commits/76d50d3a5bf0473adec685a23b83c0424881908c)

File: src/TableTransforms.jl [hunk 4]
```diff
@@ -38,6 +38,7 @@ export
   Rename,
   Identity,
   Center,
+  Coalesce,
```
[the remark this answers is not in the document]
Comment (eliascarv) [reply src/TableTransforms.jl]: Made in [76d50d3](https://github.com/JuliaML/TableTransforms.jl/pull/40/commits/76d50d3a5bf0473adec685a23b83c0424881908c)

**Review by juliohm — COMMENTED** [review]
Great, now we need to add the transform to the table in the README.

**Comment by eliascarv** [comment]
This PR is ready for a final review @juliohm!

**Review by juliohm — APPROVED** [review]

**Comment by juliohm** [comment]
Great, feel free to merge after the tests pass. 💯

**Pull request merged**